lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -1626,3 +1632,229 @@ def test_consent_required(self, mock_request):
                         kwargs=dict(course_id=course_id, discussion_id=self.discussion_id, thread_id=thread_id)),
         ):
             self.verify_consent_required(self.client, url)
+
+
+class DividedDiscussionsTestCase(CohortViewsTestCase):
```

Comment:
I created this base test class to include the create_divided_discussions method (this was moved from CohortViewsTestCase, as only these 2 moved test classes needed it).